Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -167,7 +167,7 @@ extension NSString {
                 return NSMaxRange(line.range)
             }
             let utf8View = line.content.utf8
-            let endUTF8Index = utf8View.index(utf8View.startIndex, offsetBy: diff, limitedBy: utf8View.endIndex)!
+            let endUTF8Index = utf8View.index(utf8View.startIndex, offsetBy: diff, limitedBy: utf8View.endIndex) ?? utf8View.startIndex
```

Comment:
Wouldn't it make more sense to fall back to `?? utf8View.endIndex` instead of `?? utf8View.startIndex`?

Reply:
Hi @jpsim, `utf8View.endIndex` also work works.

Reply:
I changed the code to use `utf8View.endIndex` and the tests broke so I reverted to 'utf8View.startIndex' but tests were still broken so I reverted that. Take your pick 🙂